### ISSUE TITLE
OpenJDK12 and Cellprofiler (latest) doesn't work on mac

### ISSUE THREAD

**Issue by ctr26** [CellProfiler/CellProfiler#4117]
Hi all,

Unsure where to put this as I solved my own problem.

tl;dr: Use openJDK 11 and pip3 with master cellprofiler.

From what I can tell I need to compile cellprofiler from source to be able to use cellprofiler plugins (fine).
I followed the Mac and the conda instructions for installing Cellprofiler; neither of them worked except for

`pip3 install --editable $HOME/CellProfiler`

Using the master branch, so I guess I'll be using cp4 (also fine). 

But:

`~/CellProfiler/cellprofiler master
cellprofiler ❯ cellprofiler
Could not find Java JRE compatible with x86_64 architecture
Could not find Java JRE compatible with x86_64 architecture
Could not find Java JRE compatible with x86_64 architecture
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/javabridge/jutil.py", line 114, i`

`~/CellProfiler/cellprofiler master
cellprofiler ❯ java -version
openjdk version "1.8.0_112"
OpenJDK Runtime Environment (Zulu 8.19.0.1-macosx) (build 1.8.0_112-b16)
OpenJDK 64-Bit Server VM (Zulu 8.19.0.1-macosx) (build 25.112-b16, mixed mode)`

I installed java using brew which now defaults to openjdk since oracle started login-walling java. 

`~/CellProfiler/cellprofiler master
cellprofiler ❯ pip3 install python-javabridge --upgrade
Requirement already up-to-date: python-javabridge in /usr/local/lib/python3.8/site-packages (4.0.0rc2)
Requirement already satisfied, skipping upgrade: numpy in /usr/local/lib/python3.8/site-packages (from python-javabridge) (1.19.1)`

The solution was:

`brew install openjdk@11`

Best

Craig

**Comment by bethac07** [comment]
CellProfiler should now support Java up through 14.
